runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectRunnerTest.java

```diff
@@ -577,4 +577,10 @@ public void validate() {
       return underlying.getDefaultOutputCoder();
     }
   }
+
+  @Test
+  public void testLatencyStuff() {
```

Comment:
Whoops. That was there for https://issues.apache.org/jira/browse/BEAM-2398?filter=-1